src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -107,37 +120,23 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
         )
       );
 
-      //todo tools functionality needs to be added to toolbar
-      testIf(theme !== 'refresh-toolbar')(
-        'focuses tools panel closed button when it is opened using keyboard and caused split panel to change position',
+      test(
+        'focuses tools panel closed button when it is opened and caused split panel to change position',
         setupTest(
           async page => {
-            await page.setWindowSize({ width: 1000, height: 800 });
+            await page.setWindowSize({ width: 878, height: 800 });
             await page.click(wrapper.findSplitPanel().findOpenButton().toSelector());
-            await page.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
+            if (theme !== 'refresh-toolbar') {
+              await page.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
+            } else {
+              await page.click(wrapper.findToolsToggle().toSelector());
+            }
             await expect(page.isFocused(wrapper.findToolsClose().toSelector())).resolves.toBe(true);
           },
           { pageName: 'with-split-panel', theme, mobile, splitPanelPosition: 'side' }
         )
       );
 
-      test(
-        'focuses tools panel closed button when it is opened using keyboard and caused split panel to change position in toolbar theme',
-        setupTest(
-          async page => {
-            const triggerSelector =
-              theme === 'refresh-toolbar'
-                ? wrapper.findDrawerTriggerById('slide-panel').toSelector()
-                : wrapper.findSplitPanel().findOpenButton().toSelector();
-            await page.setWindowSize({ width: 1000, height: 800 });
-            await page.click(triggerSelector);
-            await page.keys(['Tab', 'Tab', 'Enter']);
-            await expect(page.isFocused(triggerSelector)).resolves.toBe(true);
-          },
-          { pageName: 'with-split-panel', theme, mobile, splitPanelPosition: 'side' }
-        )
-      );
-
```

Comment:
This test wasn't actually testing what it was describing. This was testing split panel open/close behavior, which is already covered in other tests. Combined the changing position case with the test above.